format/mbr/mbr.jq

```diff
@@ -0,0 +1,7 @@
+def _mbr__help:
+    { notes: "Supports decoding Master Boot Record data",
```

Comment:
Looking at the generated help text maybe this is redundant? I've used this to note missing/extra features etc